api/src/test/java/org/openmrs/api/db/PatientDAOTest.java

```diff
@@ -2033,4 +2033,33 @@ public void getPatients_shouldGetNoVoidedPersonWhenVoidedFalseIsPassed() throws
 		List<Patient> patients = dao.getPatients("voided-bravo", false, 0, 11);
 		Assert.assertEquals(0, patients.size());
 	}
+	/**
```

Comment:
@karrtikiyer-tw what about adding a tests for when one adds an attribute that does not exist like "foobar" or one that is not persisted in the db. these are definitely cases that could happen and so if we have those tests we can make sure it works, when someone else later refactors your methods we make sure he doesnt break this behavior.